lib/src/validator_builder.dart

```diff
@@ -101,20 +104,20 @@ class ValidationBuilder {
   }
 
   /// Value must not be null
-  ValidationBuilder required([String message]) =>
+  ValidationBuilder required([String? message]) =>
       add((v) => (v == null || v == '') ? message ?? _locale.required() : null);
 
   /// Value length must be greater than or equal to [minLength]
-  ValidationBuilder minLength(int minLength, [String message]) => add((v) =>
-      v.length < minLength ? message ?? _locale.minLength(v, minLength) : null);
+  ValidationBuilder minLength(int minLength, [String? message]) => add((v) =>
+      v!.length < minLength ? message ?? _locale.minLength(v, minLength) : null);
```

Comment:
Could we replace the short variable names with more meaningful ones?

Like

```dart
ValidationBuilder minLength(int minLength, [String? message]) => add((value) =>
      value!.length < minLength ? message ?? _locale.minLength(value, minLength) : null);
```

Reply:
Which variables are you referring to? `minLength`?

Reply:
@TheMisir I'm referring to using `value` instead of `v`.

I also added a sample example of new one